ansible_ai_connect/ai/api/tests/test_views.py

```diff
@@ -3261,6 +3343,28 @@ def test_wca_instance_deleted(self):
             "WCA Instance has been deleted when requesting playbook generation",
         )
 
+    def test_wca_request_with_model_id_given(self):
+        self.assertion_count = 0
+        self.payload["model"] = "mymodel"
+        model_client = self.stub_wca_client(
+            200,
+        )
+
+        def get_model_id(user, organization_id, model_id):
+            self.assertEqual(model_id, "mymodel")
+            self.assertion_count += 1
+            return model_id
+
+        model_client.get_model_id = get_model_id
+        model_client.generate_playbook = lambda *args: ("playbook", "outline")
+        self.assert_test(
+            model_client,
+            HTTPStatus.OK,
+            None,
+            None,
+        )
+        self.assertGreater(self.assertion_count, 0)
```

Comment:
Same comment about not really checking the real method did what we expected. Sorry.